tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -6,7 +6,7 @@
 """
 
 import logging
-from typing import Dict, Iterator
+from typing import Dict, Iterator, Tuple, Optional
```

Comment:
linter is complaining about the order here but `Optional` is also unused: could be removed entirely.

Sorry, we have a lot of lint checks -- the code used to be a mishmash of styles and we decided being strict is better than dealing with that...